sportsref/nfl/players.py

```diff
@@ -233,6 +242,111 @@ def rushing_and_receiving(self, kind='R'):
         df = sportsref.utils.parse_table(table)
         return df
 
+    @sportsref.decorators.memoize
+    @sportsref.decorators.kind_rpb(include_type=True)
+    def defense_and_fumbles(self, kind='R'):
+        """Gets yearly defense/fumble stats for the player.
+
+        :kind: One of 'R', 'P', or 'B'. Case-insensitive; defaults to 'R'.
+        :returns: Pandas DataFrame with defense/fumble stats.
+        """
+        doc = self.get_doc()
+        table = (doc('#all_defense') if kind == 'R'
```

Comment:
Thanks for the suggestion. I changed all of the methods I wrote to incorporate this approach.